src/script/telemetry/calling/CallSetupTimings.ts

```diff
@@ -59,9 +81,9 @@ class CallSetupTimings {
     return timings;
   }
 
-  time_step(step) {
+  time_step(step: CallSetupSteps) {
     if (this[step] === 0) {
-      this[step] = window.parseInt(window.performance.now() - this.started);
+      this[step] = Math.floor(window.performance.now() - this.started);
```

Comment:
`parseInt` does a flooring after parsing the string. Since `window.performance.now()` and `this.started` are both numbers, we can't use `parseInt` anymore.

Reply:
Thanks for the explanation :+1: